pkgs/development/compilers/go/binary.nix

```diff
@@ -29,6 +29,7 @@ stdenv.mkDerivation rec {
   dontStrip = stdenv.hostPlatform.isDarwin;
 
   installPhase = ''
+    runHook preInstall
```

Comment:
`nixpkgs-hammering` will probably flag this a some point, adding them in this PR as they cause a full `go_*` rebuild.

Reply:
https://github.com/NixOS/nixpkgs/pull/125745